deb/openmediavault/usr/share/php/openmediavault/system/filesystem/filesystem.inc

```diff
@@ -895,6 +895,20 @@ class Filesystem extends \OMV\System\BlockDevice
 		$deviceFile = $mp->getDeviceFile();
 		$mngr = Backend\Manager::getInstance();
 		$backend = $mngr->getBackendById($deviceFile);
+		// if no backend found, try finding type and get backend by type
+		if (NULL == $backend) {
+			$type = "";
+			foreach ($mngr->enumerate() as $datak => $datav) {
```

Comment:
The enumerate function is too expensive in this case. On slow systems it can take ages to collect the necessary data.